### ISSUE TITLE
hyperparameterization

### ISSUE THREAD

**Issue by L-JunJie** [WangYZ1608/Self-Prompt-Tuning#6]
Hi author, can you provide the instructions for running the FGVC dataset, including the learning rate for each dataset, etc.?
